edx_exams/apps/api/v1/views.py

```diff
@@ -384,11 +394,29 @@ def get(self, request, exam_id):
 class ExamAttemptView(ExamsAPIView):
     """
     Endpoint for the ExamAttempt
-    /exams/attempt/<attempt_id>
+    /exams/attempt
 
     Supports:
+        HTTP GET: Get an attempt's status.
```

Comment:
Can we be a little more descriptive here? Maybe something like "Get the requesting user's current exam attempt", or whatever you feel is best.

Reply:
Does "HTTP GET: Fetches a user's latest in-progress attempt." work?